.vscode/launch.json

```diff
@@ -10,8 +10,8 @@
         "${file}",
         "--config",
         "jest.config.js",
-        "--no-cache",
-        "--testTimeout=2147483647"
+        "--no-cache"
+        // "--testTimeout=2147483647"
```

Comment:
this has to be reverted, which i guess just means that u need to merge `next` to this branch